src/core/utils/color/ColorValue.ts

```diff
@@ -525,10 +487,15 @@ export class ColorValue {
     );
   }
 
-  /** Create an HslColor from this ColorValue */
+  /** Get the alpha value for this ColorDef. Will be between 0-255 */
   public toHsl(): HslColor {
+    return ColorValue.toHsl(this._tbgr);
+  }
+
+  /** Create an HslColor from this ColorValue */
+  private static toHsl(tbgr: number): HslColor {
```

Comment:
Two things:
1. Probably don't want to repeat the same function name. Maybe the static one should be called `toHsvFromTbgr` or something.

2. The comments for both are wrong.
   - toHsl has nothing to do with alpha value
   - the static method says "this ColorValue", but there is no "this"
   - Also, there is no `ColorDef` anywhere